src/main/test/ApplicationUtilsTests.cpp

```diff
@@ -106,11 +106,13 @@ TEST_CASE("offline self-check works", "[applicationutils][selfcheck]")
             "client");
         catchupSimulation.catchupOffline(app, l1);
         chkConfig = app->getConfig();
-        victimBucketPath = app->getBucketManager()
-                               .getBucketList()
-                               .getLevel(0)
-                               .getCurr()
-                               ->getFilename();
+        auto filenameOp = app->getBucketManager()
+                              .getBucketList()
+                              .getLevel(0)
+                              .getCurr()
+                              ->getFilename(BucketSortOrder::SortByType);
```

Comment:
This will stop working when we transition to the new type. Maybe we need a `getPrimaryFilename` as well, for such cases?